### DISCUSSION TITLE
connection error: desc = \"transport is closing\"" when upgrading jaeger-agent from 1.22 -> 1.38

### DISCUSSION THREAD

**Discussion by logan-bobo** [jaegertracing/jaeger#5095] [Q&A]
We have recently upgraded our Jaeger agent service from `1.22` to `1.38` and in the agent and collector logs we are seeing an increased volume of the following log.

```
{
  "level": "info",
  "ts": ...,
  "caller": "grpclog/component.go:71",
  "msg": "[transport]transport: loopyWriter.run returning. connection error: desc = \"transport is closing\"",
  "system": "grpc",
  "grpc_log": true
}
```

I could not see anything in the breaking changes between our old running version and the new. Appreciate if this is just a service info level log that can be ignore and the higher volume is expected. I also saw something very similar on stack overflow - https://stackoverflow.com/questions/71520305/jaeger-collector-running-on-aws-ecs-with-grpc

Is this simply just the connection is closing or there is an actionable error? The log level of info feels this is just logging a connection closing.

**Comment by yurishkuro** [comment]
Looks harmless. Are you doing any before/after monitoring of Jaeger performance during the release?

**Reply by logan-bobo** [reply]
Thanks for getting back to me on this. Yeah we have monitoring is in place, the CPU/Memory load has stayed largely the same, coupled with seeing no HTTP errors or span failures from the agent.
